tests/testthat/test-derive_params_growth_age.R

```diff
@@ -120,7 +120,7 @@ test_that("derive_params_growth_age Test 2: Height SDS and percentile works (P50
 })
 
 ## Test 3: BMI SDS and percentile works (Z-score of -2, 0, 2, 5) ----
-test_that("derive_params_growth_age Test 3: BMI SDS and percentile works (Z-score of -2, 0, 2, 5)", {
+test_that("derive_params_growth_age Test 3: BMI SDS and percentile works (Z-score of -2, 0, 2, 5)", { # nolint
```

Comment:
Hi @zdz2101 , what is the purpose of adding "# nolint" in this line?

Reply:
Our lintr doesn't line code with more than 100 characters, so to get around it occasionally you can throw a `# nolint`